### PR TITLE
[Easy] update thumbnail with better opacity

### PR DESCRIPTION
**Pull request by sayakpaul**
Summary of the updates for https://huggingface.co/blog/simple_sdxl_optimizations:

* Updated the thumbnail to have better opacity. 
* Added acknowledgments. 
* Consistent code indentation. 

Cc: @stevhliu

### CODE REVIEW

File: simple_sdxl_optimizations.md [hunk 1]
```diff
@@ -135,10 +139,10 @@ In SDXL, a variational encoder (VAE) decodes the refined latents (predicted by t
...
 ```python
-pipe = StableDiffusionXLPipeline.from_pretrained(
-    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
-)
-pipe = pipe.to("cuda")
+pipeline = StableDiffusionXLPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-base-1.0",
+    torch_dtype=torch.float16,
+).to("cuda")
 pipe.enable_vae_slicing()
```
Comment (pcuenca) [inline simple_sdxl_optimizations.md:146]: There's a `pipe` below :)

File: simple_sdxl_optimizations.md [hunk 2]
```diff
@@ -204,9 +207,10 @@ As previously mentioned, a VAE decodes latents into images. Naturally, this step
...
 
-pipe = StableDiffusionXLPipeline.from_pretrained(
-    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
-)
+pipeline = StableDiffusionXLPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-base-1.0",
+    torch_dtype=torch.float16,
+).to("cuda")
 pipe.vae = AutoencoderTiny.from_pretrained("madebyollin/taesdxl", torch_dtype=torch.float16)
 pipe = pipe.to("cuda")
```
Comment (pcuenca) [inline simple_sdxl_optimizations.md:215]: Same: we are using `pipe` later and the `to("cuda")` is redundant now.

**Review by pcuenca — APPROVED** [review]
Awesome, thanks!

File: simple_sdxl_optimizations.md [hunk 3]
```diff
@@ -135,10 +139,10 @@ In SDXL, a variational encoder (VAE) decodes the refined latents (predicted by t
...
 
 ```python
-pipe = StableDiffusionXLPipeline.from_pretrained(
-    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
-)
-pipe = pipe.to("cuda")
+pipeline = StableDiffusionXLPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-base-1.0",
+    torch_dtype=torch.float16,
+).to("cuda")
```
Comment (sayakpaul) [inline simple_sdxl_optimizations.md:145]:
```suggestion
pipe = StableDiffusionXLPipeline.from_pretrained(
    "stabilityai/stable-diffusion-xl-base-1.0",
    torch_dtype=torch.float16,
).to("cuda")
```

File: simple_sdxl_optimizations.md (hunk 1 above)
Comment (sayakpaul) [reply simple_sdxl_optimizations.md:146]: Nice catch!

File: simple_sdxl_optimizations.md [hunk 4]
```diff
@@ -204,9 +207,10 @@ As previously mentioned, a VAE decodes latents into images. Naturally, this step
 ```python
 from diffusers import AutoencoderTiny
 
-pipe = StableDiffusionXLPipeline.from_pretrained(
-    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
-)
+pipeline = StableDiffusionXLPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-base-1.0",
+    torch_dtype=torch.float16,
+).to("cuda")
```
Comment (sayakpaul) [inline simple_sdxl_optimizations.md:213]:
```suggestion
pipe = StableDiffusionXLPipeline.from_pretrained(
    "stabilityai/stable-diffusion-xl-base-1.0",
    torch_dtype=torch.float16,
).to("cuda")
```

File: simple_sdxl_optimizations.md (hunk 2 above)
Comment (sayakpaul) [inline simple_sdxl_optimizations.md:215]:
```suggestion
pipe.vae = AutoencoderTiny.from_pretrained("madebyollin/taesdxl", torch_dtype=torch.float16)
```

**Pull request merged**